test/IRGen/type_layout_reference_storage.swift

```diff
@@ -39,10 +39,10 @@ struct ReferenceStorageTypeLayout<T, Native : C, Unknown : AnyObject> {
   // CHECK-32: store i8** getelementptr inbounds ([4 x i8*], [4 x i8*]* @type_layout_8_4_[[REF_XI:[0-9a-f]+]]_pod, i32 0, i32 0)
   unowned(unsafe) var pu:  P
   // CHECK-64: store i8** getelementptr inbounds ([4 x i8*], [4 x i8*]* @type_layout_16_8_[[WEAK_XI:[0-9a-f]+]], i32 0, i32 0)
-  // CHECK-32: store i8** getelementptr inbounds ([3 x i8*], [3 x i8*]* @type_layout_8_4_[[WEAK_XI:[0-9a-f]+]], i32 0, i32 0)
+  // CHECK-32: store i8** getelementptr inbounds ([4 x i8*], [4 x i8*]* @type_layout_8_4_[[WEAK_XI:[0-9a-f]+]], i32 0, i32 0)
   weak            var pwo: P?
   // CHECK-64: store i8** getelementptr inbounds ([4 x i8*], [4 x i8*]* @type_layout_16_8_[[WEAK_XI]], i32 0, i32 0)
-  // CHECK-32: store i8** getelementptr inbounds ([3 x i8*], [3 x i8*]* @type_layout_8_4_[[WEAK_XI]], i32 0, i32 0)
+  // CHECK-32: store i8** getelementptr inbounds ([4 x i8*], [4 x i8*]* @type_layout_8_4_[[WEAK_XI]], i32 0, i32 0)
```

Comment:
Hmm.  What caused this change?

Reply:
This change was caused by the consolidation of the extra inhabitance logic between weak and unowned. I thought you were okay with the test fallout given your comment here: https://github.com/apple/swift/pull/16237#pullrequestreview-118134931

That being said, I feel the least confident about this change. Your feedback would be appreciated.

Reply:
Oh, I understand why this changed now, thanks.  It's fine.